frontend/src/components/InputField.vue

```diff
@@ -21,7 +21,7 @@ defineProps<Props>()
       <span v-if="!!required">*</span>
     </label>
 
-    <slot :id="id" />
+    <slot :id="id"> </slot>
```

Comment:
Was that a suggestion from your IDE? That space in between though should probably go 🤔

Reply:
yes! but i put the shorthand back in :)